tutorials/btp-hana-tenantdb-xsapp-create/btp-hana-tenantdb-xsapp-create.md

```diff
@@ -0,0 +1,178 @@
+
+## Details
+### You will learn
+How to create and bind an SAP HANA tenant database to an SAP HANA XS classic application using the SAP BTP cockpit, the SAP HANA cockpit, and the SAP HANA Web-Based Development Workbench
+
+To perform the different steps in this tutorial, you use different tools. You first create a tenant database, and then create the XS classic application on the database.
+
+![Overview](intro-flow.png)
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Create a database)]
+
+In your subaccount in the SAP BTP cockpit, you create a database on an SAP HANA tenant database system.
+
+1.  In the SAP BTP cockpit, navigate to a subaccount. For more information, see [Navigate in the Cockpit](https://help.sap.com/viewer/ea72206b834e4ace9cd834feed6c0e09/Cloud/en-US/fdeff7e68f64496eb8a1fb31f6a08b73.html).
+
+2.  Choose **SAP HANA / SAP ASE** > **Databases & Schemas** in the navigation area.
+
+    All databases available in the selected account are listed with their ID, type, version, and related database system.
+
+    **Tip**
+    > To view the details of a database, for example, its state and the number of existing bindings, select a database in the list and click the link on its name. On the overview of the database, you can perform further actions, for example, delete the database.
+
+3.  From the **Databases & Schemas** page, choose **New**.
+
+4.  Enter the required data:
+
+    |  Property     | Value     | Description
+    |  :------------- | :------------- | :-------------
+    |  Database ID    | Example: **`tutorial`**    |   HANA database ID
+    |  Database System    | Example: **`mdc1 (HANAMDC)`**    |  An SAP HANA tenant database system. **`mdc1mdc1`** corresponds to the database system on which you create the database.          
+    |  SYSTEM User Password    |     |  The password for the SYSTEM user of the database.   The SYSTEM user is a preconfigured database superuser with irrevocable system privileges, such as the ability to create other database users, access system tables, and so on. A database-specific SYSTEM user exists in every database of a tenant database system.
+
+5.  Choose **Create**.
+
+6.  The **Events** page shows the progress of the database creation. Wait until the tenant database is in state **Started**.
+
+7.  (Optional) To view the details of the new database, choose **Overview** in the navigation area and select the database in the list. Verify that the status **STARTED** is displayed.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Create a database user)]
+
+Create a new database user in the SAP HANA cockpit and assign the user the required permissions for working with the SAP HANA Web-based Development Workbench.
+
+You'll perform all subsequent activities with this new user.
+
+> You've specified a password for the SYSTEM user when you created an SAP HANA tenant database. You now use the SYSTEM user to log on to SAP HANA cockpit and create your own database administration user.
+
+**Caution**
```

Comment:
This should be part of the note.

Reply:
Same as above. Please provide guidance on how this can be formatted.